src/sc_client/constants/sc_type/__init__.py

```diff
@@ -0,0 +1,127 @@
+from sc_client.constants.sc_type import bitmasks as b
+from sc_client.constants.sc_type.sc_type import ScType
+
+UNKNOWN = ScType()
+
+# sc-elements
+NODE = ScType(b.SC_TYPE_NODE)
+CONNECTOR = ScType(b.SC_TYPE_CONNECTOR)
+COMMON_EDGE = ScType(b.SC_TYPE_COMMON_EDGE)
+ARC = ScType(b.SC_TYPE_ARC)
+COMMON_ARC = ScType(b.SC_TYPE_COMMON_ARC)
+MEMBERSHIP_ARC = ScType(b.SC_TYPE_MEMBERSHIP_ARC)
+
+# constancy
+CONST = ScType(b.SC_TYPE_CONST)
+VAR = ScType(b.SC_TYPE_VAR)
+
+CONST_NODE = ScType(b.SC_TYPE_CONST | b.SC_TYPE_NODE)
+VAR_NODE = ScType(b.SC_TYPE_VAR | b.SC_TYPE_NODE)
+CONST_CONNECTOR = ScType(b.SC_TYPE_CONST | b.SC_TYPE_CONNECTOR)
+VAR_CONNECTOR = ScType(b.SC_TYPE_VAR | b.SC_TYPE_CONNECTOR)
+CONST_COMMON_EDGE = ScType(b.SC_TYPE_CONST | b.SC_TYPE_COMMON_EDGE)
+VAR_COMMON_EDGE = ScType(b.SC_TYPE_VAR | b.SC_TYPE_COMMON_EDGE)
+CONST_ARC = ScType(b.SC_TYPE_CONST | b.SC_TYPE_ARC)
+VAR_ARC = ScType(b.SC_TYPE_VAR | b.SC_TYPE_ARC)
+CONST_COMMON_ARC = ScType(b.SC_TYPE_CONST | b.SC_TYPE_COMMON_ARC)
+VAR_COMMON_ARC = ScType(b.SC_TYPE_VAR | b.SC_TYPE_COMMON_ARC)
+CONST_MEMBERSHIP_ARC = ScType(b.SC_TYPE_CONST | b.SC_TYPE_MEMBERSHIP_ARC)
+VAR_MEMBERSHIP_ARC = ScType(b.SC_TYPE_VAR | b.SC_TYPE_MEMBERSHIP_ARC)
+
+# permanency
+PERM_ARC = ScType(b.SC_TYPE_PERM_ARC)
+TEMP_ARC = ScType(b.SC_TYPE_TEMP_ARC)
+
+CONST_PERM_ARC = ScType(b.SC_TYPE_CONST | b.SC_TYPE_PERM_ARC)
+VAR_PERM_ARC = ScType(b.SC_TYPE_VAR | b.SC_TYPE_PERM_ARC)
+CONST_TEMP_ARC = ScType(b.SC_TYPE_CONST | b.SC_TYPE_TEMP_ARC)
+VAR_TEMP_ARC = ScType(b.SC_TYPE_VAR | b.SC_TYPE_TEMP_ARC)
+
+# actuality
+ACTUAL_TEMP_ARC = ScType(b.SC_TYPE_ACTUAL_ARC | b.SC_TYPE_TEMP_ARC)
+INACTUAL_TEMP_ARC = ScType(b.SC_TYPE_INACTUAL_ARC | b.SC_TYPE_TEMP_ARC)
+
+CONST_ACTUAL_TEMP_ARC = ScType(b.SC_TYPE_CONST | b.SC_TYPE_ACTUAL_ARC | b.SC_TYPE_TEMP_ARC)
+VAR_ACTUAL_TEMP_ARC = ScType(b.SC_TYPE_VAR | b.SC_TYPE_ACTUAL_ARC | b.SC_TYPE_TEMP_ARC)
+CONST_INACTUAL_TEMP_ARC = ScType(b.SC_TYPE_CONST | b.SC_TYPE_INACTUAL_ARC | b.SC_TYPE_TEMP_ARC)
+VAR_INACTUAL_TEMP_ARC = ScType(b.SC_TYPE_VAR | b.SC_TYPE_INACTUAL_ARC | b.SC_TYPE_TEMP_ARC)
+
+# positivity
+POS_ARC = ScType(b.SC_TYPE_POS_ARC)
+NEG_ARC = ScType(b.SC_TYPE_NEG_ARC)
+
+# fuzzy sc-arcs
```

Comment:
```suggestion
# fuzziness
```

Reply:
and place it after // positive sc-arcs section to follow common style when new subtype is introduced and then it is combined with all possible types

Reply:
I'm not quite sure where you want to move it

Reply:
before line that starts with '# fuzzy sc-arcs'

Reply:
I still don't understand what you want me to do.

Reply:
Fixed